doc/source/release/1.20.0-notes.rst

```diff
@@ -253,6 +253,18 @@ library.
 Compatibility notes
 ===================
 
+``isinstance(dtype, np.dtype)`` and not ``type(dtype) is not np.dtype``
+-----------------------------------------------------------------------
+NumPy dtypes are not direct instances of ``np.dtype`` anymore.  Code that
+may have used ``type(dtype) is np.dtype`` will always return ``False`` and
+must be updated to use the correct version ``isinstance(dtype, np.dtype)``.
+
+This change also affects the C-side macro ``PyArray_DescrCheck`` if compiled
+against a NumPy older than 1.16.6. If code uses this macro and wishes to
+compile against an older version of NumPy, it must replace the macro.
+(See also the `The ``PyArray_DescrCheck`` macro is modified`_).
```

Comment:
Not quite sure this link will show correctly, so going to check when CI is done...

Reply:
OK, I changed it to link to the "C API changes" section instead which works (I guess the backticks trip sphinx), that seems good enough.